docs/metrics.md

```diff
@@ -81,6 +81,9 @@ The current available memory of the VM containers based on the rss. Type: Gauge.
 ### kubevirt_vm_container_free_memory_bytes_based_on_working_set_bytes
 The current available memory of the VM containers based on the working set. Type: Gauge.
 
+### kubevirt_vm_created_total
+Amount of VMs created, broken down by namespace. Type: Counter.
```

Comment:
Created since when?
Since instal, update, controller pod restart, phase of the moon :) …

Reply:
we can't obviously have the counter track VMs from previous versions, but other than that Prometheus takes care of summing all counter values.

maybe this is a good opportunity to think once again about more metrics metadata, release lifecycle, version when it was added, when they are deprecated, etc... (would also help with the discussion in https://groups.google.com/g/kubevirt-dev/c/7p3q5Lo71hs/m/jIJGNJpdAwAJ)